R/pkg/R/SQLContext.R

```diff
@@ -254,6 +301,7 @@ jsonFile <- function(sqlContext, path) {
 #' df <- jsonRDD(sqlContext, rdd)
 #'}
 
+# TODO: remove - this method is no longer exported
```

Comment:
opened SPARK-15545
one reason to not remove this right away is because we have been talking about calling internal methods via `SparkR:::jsonRDD` for this and other RDD methods.
we have .Deprecated() here though for a warning